systers_portal/users/models.py

```diff
@@ -189,3 +189,22 @@ def create_systers_user(sender, instance, created, **kwargs):
         if instance is not None:
             systers_user = SystersUser(user=instance)
             systers_user.save()
+
+
+class UserSetting(models.Model):
+    user = models.OneToOneField(SystersUser, on_delete=models.CASCADE)
+    weekly_digest = \
+        models.BooleanField(default=True,
```

Comment:
This is weird formatting :smile:

Reply:
For the flake8 to pass. You have any suggestions on this?

Reply:
```suggestion
        models.BooleanField(
            default=True, verbose_name="Receive Weekly Digests from Communities")
```

Reply:
@SanketDG Done